spec/formatting.md

```diff
@@ -10,14 +10,18 @@ the successor to ICU MessageFormat, henceforth called ICU MessageFormat 1.0.
 
 ## Variable Resolution
 
-To resolve the value of a Variable,
-its Name is used to identify either a local variable,
+To resolve the value of a _variable_,
+its _name_ is used to identify either a local variable,
 or a variable defined elsewhere.
-If a local variable and an externally defined one use the same name,
-the local variable takes precedence.
 
-It is an error for a local variable definition to
-refer to a local variable that's defined after it in the message.
+If more than one _declaration_ binds a value to the same _name_,
```

Comment:
Let's avoid the word "binds", as requested by @mihnita in https://github.com/unicode-org/message-format-wg/issues/310#issuecomment-1596280816.

```suggestion
If the left-hand side _variable_ of more than one _declaration_ uses the same _name_,
```